website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -19,23 +19,25 @@ Use the [Account API](/terraform/cloud-docs/api-docs/account) to get account det
 
 <!-- BEGIN: TFC:only -->
 
-## Log In With Your HashiCorp Cloud Platform Account
+## Log in with a HashiCorp Cloud Platform account
 
 We recommend using a [HashiCorp Cloud Platform (HCP)](https://portal.cloud.hashicorp.com/sign-up) account to log in to HCP Terraform. Your HCP Account grants access to every HashiCorp product and the Terraform Registry. If you use an HCP Account, you manage account settings like multi-factor authentication and password resets from within HCP instead of the HCP Terraform UI.
 
 To log in with your HCP account, go to the [Sign In to HCP Terraform](https://app.terraform.io/) page and click **Continue with HCP account**. HCP Terraform may ask if you want to link your account.
 
-### Linked HCP and HCP Terraform Accounts
+### Linking HCP and HCP Terraform accounts
 
-The first time you log in with your HCP credentials, HCP Terraform searches for existing HCP Terraform accounts with the same email address. If there is an unlinked account, HCP Terraform asks if you want to link it to your HCP account. If there is no existing HCP Terraform account with that email address, HCP Terraform creates one and automatically links it to your HCP account.
+The first time you log in with your HCP credentials, HCP Terraform searches for an existing HCP Terraform account with the same email address. If you have an unlinked account, HCP Terraform asks if you want to link it to your HCP account. Otherwise, if no account matches your HCP account's email address, HCP Terraform creates and automatically links a new HCP Terraform account to your HCP account.
 
-After you link your HCP and HCP Terraform accounts, you can only log in with your HCP credentials. You must unlink the accounts to resume logging in with your HCP Terraform username and password.
+!> **Compatibility warning**: After linking your HCP and HCP Terraform accounts, you can only log in with your HCP credentials. We do not recommend linking your account if you use an SSO provider to log in to HCP Terraform.
```

Comment:
```suggestion
> **Note**: After linking your HCP and HCP Terraform accounts, you can only log in with your HCP credentials. We do not recommend linking your account if you use an SSO provider to log in to HCP Terraform because linking your account may conflict with your existing SSO configuration. 
```